web/src/lib/nostr/utils.js

```diff
@@ -82,3 +75,45 @@ export async function queryNip05(fullname) {
 
     return false;
 }
+
+export function filterTags(tagsArray, tagToFilter) {
+    return tagsArray.filter(t => {
+        return t[0] === tagToFilter;
+    });
+}
+
+export function findMarkerInTag(eventBeingRepliedToTagsE, tagType, marker) {
+    let found = false;
+
+    eventBeingRepliedToTagsE.forEach(tag => {
```

Comment:
Since you check the tagType here, the parameter should not be called `...TagsE` - right? Because it could be any tag.

Reply:
Yes, your're right. It will be uses for non-E tags.